src/openforms/appointments/contrib/qmatic/admin.py

```diff
@@ -1,10 +1,31 @@
+from django import forms
 from django.contrib import admin
 
 from solo.admin import SingletonModelAdmin
 
+from openforms.utils.form_fields import (
+    CheckboxChoicesArrayField,
+    get_arrayfield_choices,
+)
+
 from .models import QmaticConfig
 
 
+class RequiredCustomerFieldsField(CheckboxChoicesArrayField):
+    @staticmethod
+    def get_field_choices():
+        return get_arrayfield_choices(QmaticConfig, "required_customer_fields")
+
+
+class QmaticConfigForm(forms.ModelForm):
+    class Meta:
+        model = QmaticConfig
+        fields = "__all__"
```

Comment:
because it's a form for the django model admin, where we apply this convention. Usually we want to expose all model fields that exist (or are added in the future) in the admin, and this avoids forgetting to do this.